patch/backend/app/engine/reranker.py

```diff
@@ -0,0 +1,30 @@
+import os
+
+from llama_index.core.postprocessor.types import BaseNodePostprocessor
+
+
+def get_cohere_reranker():
+    from llama_index.postprocessor.cohere_rerank import CohereRerank
+
+    api_key = os.getenv("COHERE_API_KEY")
+    top_k = int(os.getenv("TOP_K", "3"))
```

Comment:
redundant with engine - centralize?

Reply:
Let me check to see is it able to centralize, but the original idea was the engine use it for both retrieval while this is for reranker, we just share the TOP_K as the expected number of result - no matter the reranker is enable or not.

Reply:
good point, but then we can do this?
```suggestion
    top_k = int(os.getenv("TOP_K", DEFAULT_TOP_K))
```